source/_integrations/tuya.markdown

```diff
@@ -78,21 +79,25 @@ Link devices by your app account and copy the **Device ID** in the **Device List
 **1.** Navigate to the **Devices** tab.
 
 **2.** Click **Link Tuya App Account** > **Add App Account**.
+
 ![](/images/integrations/tuya/image_004.png)
 
 **3.** Scan the QR code that appears using the **Tuya Smart** app or **Smart Life** app.
+
 ![](/images/integrations/tuya/image_005.png)
 
 **4.** Click **Confirm** on your **Tuya Smart** or **Smart Life** app.
 
-**5.** Navigate to the **Device List** tab. You can find the devices that have been added through your **Tuya Smart** or **Smart Life** account.   
+**5.** Navigate to the **All Devices** tab. You can find the devices that have been added through your **Tuya Smart** or **Smart Life** account.
+
 ![](/images/integrations/tuya/image_006.png)
 
 ### Install the Tuya Home Assistant integration
 
 #### Install the Tuya integration
 
 In **Configuration** > **Integrations** > **ADD INTEGRATION**, search for the keyword **Tuya** and select it for installation.
```

Comment:
The entire "Install the Tuya Home Assistant integration" section should be removed, it's already documented at the top of the document

Reply:
What about the table with the description of the fields? Kill it or move it to the beginning?

Reply:
Those are fine

Reply:
Ok moved credentials table up. Made no sense to be at the bottom after removing the integration steps. Rmoved a heading there and minor other changes.